### DISCUSSION TITLE
RTMP for Steam Broadcast

### DISCUSSION THREAD

**Discussion by Sebedai** [datarhei/restreamer#501] [Q&A] [answered]
Hello, 
I would like to send a stream to Steam Broadcast. 
https://partner.steamgames.com/doc/store/broadcast/setting_up

For this I tried to use the service RTMP. As address I tried "rtmp://ingest-rtmp.broadcast.steamcontent.com/app/MyToken". Unfortunately without success. 

If I enter the as address rtmp://ingest-rtmp.broadcast.steamcontent.com/app in OBS (Service: Custom) and specify the token in the separate token input field, the stream works. 

How do I have to enter the link to the RTMP ? Is there alternatively like in OBS the possibility to specify the address and the token separately ?

Thanks for the help

**Comment by svenerbeck** [comment, marked as answer]
Hello @Sebedai 
We have converted this into an issue. Please look here: 
https://github.com/datarhei/restreamer/issues/504

Cheerio
Sven
